File: 0438/438.go
func findAllAnagramsInAString(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findAllAnagramsInAString(root.Left)
	r := findAllAnagramsInAString(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 0438/438.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var findAllAnagramsInAString = function(root) {
    if (!root) return 0;
    return 1 + Math.max(findAllAnagramsInAString(root.left), findAllAnagramsInAString(root.right));
};